e2e/scripts/generate-project.js

```diff
@@ -54,33 +59,34 @@ const main = async (opts) => {
 
 // Define the program with description and arguments
 program
-  .description(
-    "Generate a retail-react-app project using the key <project-key> or the JSON <project-config>"
-  )
-  .option("--project-key <key>", "Project key", (value) => {
-    const validKeys = [
-      "retail-app-demo",
-      "retail-app-private-client",
-    ];
-    if (!validKeys.includes(value)) {
-      throw new Error("Invalid project key.");
-    }
-    return value;
-  })
-  .option(
-    "--project-config <config>",
-    "Project config as JSON string",
-    (value) => {
-      try {
-        return JSON.parse(value);
-      } catch (e) {
-        throw new Error("Invalid JSON string.");
-      }
-    }
-  )
-  .action((options) => {
-    // Call the main function with parsed options
-    main(options);
-  });
+    .description(
+        'Generate a retail-react-app project using the key <project-key> or the JSON <project-config>'
+    )
+    .option('--project-key <key>', 'Project key', (value) => {
+        const validKeys = [
+            'retail-app-demo',
+            'retail-app-ext',
+            'retail-app-no-ext',
+            'retail-app-private-client',
+            'retail-react-app-bug-bounty',
+            'retail-react-app-demo-site'
+        ]
```

Comment:
Are we testing all of these presets?

Reply:
To my knowledge, yes. `retail-react-app-demo-site` is something we should rename, but the preset I believe is valid

Reply:
I've replaced / fixed / migrated / deleted all strings with `retail-react-app` now